packages/cart/src/services/cart-module.ts

```diff
@@ -1100,4 +1111,355 @@ export default class CartModuleService implements ICartModuleService {
 
     await this.shippingMethodAdjustmentService_.delete(ids, sharedContext)
   }
+
+  @InjectManager("baseRepository_")
+  async listLineItemTaxLines(
+    filters: CartTypes.FilterableLineItemTaxLineProps = {},
+    config: FindConfig<CartTypes.LineItemTaxLineDTO> = {},
+    @MedusaContext() sharedContext: Context = {}
+  ) {
+    const taxLines = await this.lineItemTaxLineService_.list(
+      filters,
+      config,
+      sharedContext
+    )
+
+    return await this.baseRepository_.serialize<CartTypes.LineItemTaxLineDTO[]>(
+      taxLines,
+      {
+        populate: true,
+      }
+    )
+  }
+
+  addLineItemTaxLines(
+    taxLines: CartTypes.CreateLineItemTaxLineDTO[]
+  ): Promise<CartTypes.LineItemTaxLineDTO[]>
+  addLineItemTaxLines(
+    taxLine: CartTypes.CreateLineItemTaxLineDTO
+  ): Promise<CartTypes.LineItemTaxLineDTO>
+  addLineItemTaxLines(
+    cartId: string,
+    taxLines: CartTypes.CreateLineItemTaxLineDTO[],
+    sharedContext?: Context
+  ): Promise<CartTypes.LineItemTaxLineDTO[]>
+
+  @InjectTransactionManager("baseRepository_")
+  async addLineItemTaxLines(
+    cartIdOrData:
+      | string
+      | CartTypes.CreateLineItemTaxLineDTO[]
+      | CartTypes.CreateLineItemTaxLineDTO,
+    taxLines?: CartTypes.CreateLineItemTaxLineDTO[],
+    @MedusaContext() sharedContext: Context = {}
+  ): Promise<CartTypes.LineItemTaxLineDTO[] | CartTypes.LineItemTaxLineDTO> {
+    let addedTaxLines: LineItemTaxLine[] = []
+    if (isString(cartIdOrData)) {
+      // existence check
+      await this.retrieve(cartIdOrData, { select: ["id"] }, sharedContext)
+
+      addedTaxLines = await this.lineItemTaxLineService_.create(
+        taxLines as CartTypes.CreateLineItemTaxLineDTO[],
+        sharedContext
+      )
+    } else {
+      const data = Array.isArray(cartIdOrData) ? cartIdOrData : [cartIdOrData]
+
+      addedTaxLines = await this.lineItemTaxLineService_.create(
+        data as CartTypes.CreateLineItemTaxLineDTO[],
+        sharedContext
+      )
+    }
+
+    if (isObject(cartIdOrData)) {
+      return await this.baseRepository_.serialize<CartTypes.LineItemTaxLineDTO>(
+        addedTaxLines[0],
+        {
+          populate: true,
+        }
+      )
+    }
+
+    return await this.baseRepository_.serialize<CartTypes.LineItemTaxLineDTO[]>(
+      addedTaxLines,
+      {
+        populate: true,
+      }
+    )
```

Comment:
suggestion(non-blocking):

```suggestion
    const serialized = await this.baseRepository_.serialize<CartTypes.LineItemTaxLineDTO[]>(
      addedTaxLines,
      { populate: true }
    )
    if (isObject(cartIdOrData)) {
      return serialized[0]
    }
    return serialized
```